bin/node/runtime/pangolin/src/pallets/system.rs

```diff
@@ -24,34 +27,17 @@ const AVERAGE_ON_INITIALIZE_RATIO: Perbill = Perbill::from_percent(25);
 const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
 /// We allow for 2 seconds of compute with a 6 second average block time.
 const MAXIMUM_BLOCK_WEIGHT: Weight = 2 * WEIGHT_PER_SECOND;
+
 static_assertions::const_assert!(
 	NORMAL_DISPATCH_RATIO.deconstruct() >= AVERAGE_ON_INITIALIZE_RATIO.deconstruct()
 );
+
 frame_support::parameter_types! {
 	pub const BlockHashCount: BlockNumber = 2400;
 	pub const Version: RuntimeVersion = VERSION;
-	pub RuntimeBlockLength: BlockLength =
-		BlockLength::max_with_normal_ratio(5 * 1024 * 1024, NORMAL_DISPATCH_RATIO);
-	pub RuntimeBlockWeights: BlockWeights = BlockWeights::builder()
-		.base_block(BlockExecutionWeight::get())
-		.for_class(DispatchClass::all(), |weights| {
-			weights.base_extrinsic = ExtrinsicBaseWeight::get();
-		})
-		.for_class(DispatchClass::Normal, |weights| {
-			weights.max_total = Some(NORMAL_DISPATCH_RATIO * MAXIMUM_BLOCK_WEIGHT);
-		})
-		.for_class(DispatchClass::Operational, |weights| {
-			weights.max_total = Some(MAXIMUM_BLOCK_WEIGHT);
-			// Operational transactions have some extra reserved space, so that they
-			// are included even if block reached `MAXIMUM_BLOCK_WEIGHT`.
-			weights.reserved = Some(
-				MAXIMUM_BLOCK_WEIGHT - NORMAL_DISPATCH_RATIO * MAXIMUM_BLOCK_WEIGHT
-			);
-		})
-		.avg_block_initialization(AVERAGE_ON_INITIALIZE_RATIO)
-		.build_or_panic();
```

Comment:
Why these are removed? Not used or due to substrate upgrade?

Reply:
See the discussion above

Reply:
Also some discussion in slack

```
Xavier Lau:hatching_chick:  2 days ago
我认为 runtime 就是应该直接依赖的，primitives 里的东西基本都是不动的链类型而 runtime 这些会经常调整
Darwinia 和 Crab 用的是同一份 primitives 但是 runtime 配置各有不同
white_check_mark
eyes
clap




Yalin Cai  2 days ago
Hmm 这样也是可以, 我的想法是 如果单纯只是计算个费用 没必要把整个 runtime 都引用过去. 如果 primitives 给 darwinia crab 这些共用的话, 现在放到 primitives 里面的很多东西都可以不用了 还原之前的样子 然后其他地方都直接引用 runtime

Yalin Cai  2 days ago
暂时先这样, 等我调试完毕后 我再都迁移回 runtime, 等调试完也相对稳定一些
```